src/openms/source/CONCEPT/QCMBRalignment.cpp

```diff
@@ -11,90 +11,165 @@ QCMBRalignment::~QCMBRalignment()
 
 }
 
-int QCMBRalignment::MBRAlignment(MetricMap& raws, MetricMap& outSeq, MetricMap& outCorrRT, MetricMap& outOrigRT, MetricMap& outSpectra) const
+int QCMBRalignment::MBRAlignment(MzTab& mztab) const
 {
-      vector<String> rawFiles;
-  	  vector<String> sequences;
-  	  vector<float> originalRTs;
-  	  vector<float> correctedRTs;
-  	  vector<String> spectra;
+  vector<String> rawFiles;
+  vector<String> sequences;
+  vector<float> originalRTs;
+  vector<float> correctedRTs;
+  vector<String> spectra;
   	  
-  	  vector<FeatureMap> maps;
+  vector<FeatureMap> maps;
   	  
-  	  for(vector<pair<String,FeatureMap>>::const_iterator it = FeatMap_.begin();it!=FeatMap_.end();++it){
-        if(it->first=="MapRTTransformer"){maps.push_back(it->second);}
-      }
+  for(vector<pair<String,FeatureMap>>::const_iterator it = feat_map_.begin();it!=feat_map_.end();++it)
+  {
+    if(it->first=="MapRTTransformer"){maps.push_back(it->second);}
```

Comment:
not required. The FeatureMaps should be directly passed in the Constructor. The Metric must be able to trust that they are the correct ones.